src/Claim.ts

```diff
@@ -72,3 +72,9 @@ export function createClaim(privateKey: string, type: ClaimType, attributes: Cla
     signature,
   }
 }
+
+export function isValidClaim(claim = {}): boolean {
+  if (!isClaim(claim)) return false
```

Comment:
Don't you prefer the functional approach? `return isClaim(claim) && isValidSignature(claim) && getClaimId(claim) === claim.id`?

Reply:
Yup, that looks better, I forgot to have us go back and refactor that ;)

Reply:
I had to do it slightly different. IsClaim returns undefined for object.id(and more keys) if it is not on the object that we pass in. 
`if (!isClaim(claim)) return false`
`return isValidSignature(claim) && getClaimId(claim) === claim.id`

Reply:
Hmm I'm not sure I understand. `isClaim(claim)` can return `undefined`? If so, it's still a falsey value, so it'll evaluate to true/false

Reply:
It was making the notClaimObject test return undefined whenever I tried it that way.

Reply:
@jwicks31 try a double _not_ operator to coerce it to a boolean value. `!!isClaim(clam)``